client/src/components/App.jsx

```diff
@@ -36,13 +42,18 @@ class App extends React.Component {
   };
 
   render() {
+    const { hostName, city, country, description, gettingAround, guidebookId, lat, lng } = this.state;
     return (
       <div className="app">
         <h2>The neighborhood</h2>
-        <p>{this.state.hostName}'s place is located in {this.state.city}, {this.state.country}.</p>
-        <p>{this.state.description}</p>
-        <p>{this.state.gettingAround}</p>
-        <Details guidebookId={this.state.guidebookId} />
+        <p>{hostName}'s place is located in {city}, {country}.</p>
+        <p>{description}</p>
+        <p>{gettingAround}</p>
+
+          <Details guidebookId={guidebookId} gettingAround={gettingAround} />
+        <div  className="NeighborHoodMap">
+          <NeighborHoodMap lat={lat} lng={lng} />
```

Comment:
The components you have on line 53 and 55 aren't nested so no need to indent them here - put them on the same line as the `<p>` tags. You could even remove the white space on line 52 unless there was a reason you wanted to keep it there.